components/tooltip/tooltip.js

```diff
@@ -890,6 +886,11 @@ class Tooltip extends RtlMixin(LitElement) {
 		clearTimeout(this._hoverTimeout);
 		clearTimeout(this._longPressTimeout);
 		if (newValue) {
+			if (!this.forceShow) {
```

Comment:
The code works fine without this condition, but without it, `activeTooltip` can be assigned to a tooltip with `force-show` even though those tooltips cannot be hidden. So this just ensures that we don't have that weird state, and it makes it an explicit consideration in the code.

It would be theoretically possible to have tooltip collisions when using `force-show`. That attribute was (and still is) incompatible with the idea of only showing one tooltip at a time.